quickstarts/delphix/dashboards/my-dashboard.json

```
@@ -0,0 +1,135 @@
+{
+  "name": "Delphix Dashboard",
+  "description": null,
+  "pages": [
+    {
+      "name": "Delphix Dashboard",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "dSources",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 2909314,
```

This can cause issues when a user has a different account id, please set it to 0.
```suggestion
                "accountId": 0,
```